layouts/community/list.html

```diff
@@ -4,9 +4,9 @@
 <section class="row td-box td-box--1 position-relative td-box--gradient td-box--height-auto">
 	<div class="container text-center td-arrow-down">
 		<span class="h4 mb-0">
-<h1>Join the {{ .Site.Title }} community</h1>
+<h1>{{ T "community_join" }} {{ .Site.Title }} {{ T "community" }}</h1>
 
-<p>{{ .Site.Title }} is an open source project that anyone in the community can use, improve, and enjoy. We'd love you to join us! Here's a few ways to find out what's happening and get involved.
+<p>{{ .Site.Title }} {{ T "community_introduce" }}
```

Comment:
Again, I don't like the way translations are implemented here: Passing the . context when calling T seems the way to go here, too.